spec/workflow/intrinsic_function_spec.rb

```diff
@@ -450,6 +450,13 @@
       end
     end
 
+    describe "States.JsonToString" do
+      it "fetches values from the input" do
+        result = described_class.value("States.JsonToString($.input)", {"context" => {"baz" => "qux"}}, {"input" => {"foo" => "bar"}})
```

Comment:
The context is not needed for the test, so it visually compicates the test, so let's get rid of it and replace with a hardcoded `{}` like we do in other specs.